src/Microsoft.PowerShell.Commands.Utility/commands/utility/Join-String.cs

```diff
@@ -160,79 +160,109 @@ protected override void EndProcessing()
         }
     }
 
-    [SuppressMessage(
-        "Microsoft.Performance",
-        "CA1812:AvoidUninstantiatedInternalClasses",
-        Justification = "Class is instantiated through late-bound reflection")]
-    internal class JoinItemCompleter : IArgumentCompleter
+    /// <summary>
+    /// Provides completion for the Separator parameter of the Join-String cmdlet.
+    /// </summary>
+    public class SeparatorArgumentCompleter : IArgumentCompleter
```

Comment:
sealed?

Reply:
Fixed in de63820f08c84cdaf7cfb3d9bef341c3cd1e5aa9